<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerSearch</class>
 <widget class="QWidget" name="CustomerSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tvCustomers">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>501</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>131</width>
     <height>332</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblContractNumber">
      <property name="text">
       <string>№ договора</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lntContractNumber"/>
    </item>
    <item>
     <widget class="QLabel" name="lblPhoneNumber">
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lntPhoneNumber"/>
    </item>
    <item>
     <widget class="QLabel" name="lblLastName">
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lntLName"/>
    </item>
    <item>
     <widget class="QLabel" name="lblLocality">
      <property name="text">
       <string>Нас. пункт</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lntLocality"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Улица</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lntStreet"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Дом</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lntHouse"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>188</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="cbxNoCustomer">
      <property name="text">
       <string>Не клиент</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbtnCreateTask">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
